examples/elasticsearch/vagga.yaml

```diff
@@ -0,0 +1,27 @@
+containers:
+  elasticsearch:
+    setup:
+    - !UbuntuRelease { version: 15.10 }
```

Comment:
This way is deprecated in vagga 0.6. You should use `- !Ubuntu xenial` or `- !Ubuntu wily` if you want 15.10 specifically.